android/src/main/kotlin/id/oddbit/flutter/facebook_app_events/FacebookAppEventsPlugin.kt

```diff
@@ -47,9 +47,9 @@ class FacebookAppEventsPlugin: FlutterPlugin, MethodCallHandler {
       "getApplicationId" -> handleGetApplicationId(call, result)
       "logEvent" -> handleLogEvent(call, result)
       "logPushNotificationOpen" -> handlePushNotificationOpen(call, result)
-      "setUserData" -> handleSetUserData(call, result)
+      //"setUserData" -> handleSetUserData(call, result)
       "setUserID" -> handleSetUserId(call, result)
-      "updateUserProperties" -> handleUpdateUserProperties(call, result)
```

Comment:
1. Please clarify the removal of the functions
2. Don't comment out code that is not needed, delete it.